src/site/stages/build/process-cms-exports/schemas/transformed/node-event.js

```diff
@@ -0,0 +1,131 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-event'] },
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['event'] },
+    title: { type: 'string' },
+    entityUrl: {
+      // Probably should pull this out into a common schema
+      type: 'object',
+      properties: {
+        breadcrumb: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              url: {
+                type: 'object',
+                properties: {
+                  path: { type: 'string' },
+                  routed: { type: 'boolean' },
+                },
+                required: ['path', 'routed'],
+              },
+              text: { type: 'string' },
+            },
+            required: ['url', 'text'],
+          },
+        },
+        path: { type: 'string' },
+      },
+      required: ['breadcrumb', 'path'],
+    },
+    entityMetaTags: {
+      // Probably should be a common schema...except it's got
+      // __typename instead of type, so it's different.
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          __typename: { type: 'string' },
+          key: { type: 'string' },
+          value: { type: 'string' },
+        },
+      },
+    },
+    entityPublished: { type: 'boolean' },
+    changed: { type: 'number' },
+    uid: {
+      type: 'object',
+      properties: {
+        targetId: { type: 'number' },
+        entity: {
+          type: 'object',
+          properties: {
+            name: { type: 'string' },
+            timezone: { type: ['null'] }, // All the exmaples are null
```

Comment:
- The only reason we need to create an entity transformer for `user` is because of the `node-event` entity (as far as I know)
- The only time the `uid` is used in the templates is to grab the `uid.entity.timezone`
- Every timezone I found was `null` :unamused: